docs/reference/mapping/params/analyzer.asciidoc

```diff
@@ -1,81 +1,17 @@
 [[analyzer]]
 === `analyzer`
 
-The values of <<mapping-index,`analyzed`>> string fields are passed through an
-<<analysis,analyzer>> to convert the string into a stream of _tokens_ or
-_terms_.  For instance, the string `"The quick Brown Foxes."` may, depending
-on which analyzer is used,  be analyzed to the tokens: `quick`, `brown`,
-`fox`.  These are the actual terms that are indexed for the field, which makes
-it possible to search efficiently for individual words _within_  big blobs of
-text.
```

Comment:
This is now covered at in the [Analysis overview][0]. I think we should avoid repeating it here.


[0]: https://www.elastic.co/guide/en/elasticsearch/reference/master/analysis-overview.html.